sled-agent/src/support_bundle/mod.rs

```diff
@@ -0,0 +1,6 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+pub mod queries;
```

Comment:
FYI @papertigers - we both used the `support_bundle.rs` module, so I split your stuff into "queries" and my stuff into "storage". Hopefully the re-organization makes sense?